.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+       - image: circleci/ruby:2.4.1-node-browsers
+       - image: luckyj5/splunk-circleci-hec-720:latest
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run:
+          name: install dependencies
+          command: |
+            gem install bundler
+            bundle install --jobs=4 --retry=3 --path vendor/bundler
+
+      - run:
+          name: builder
+          command: |
+            rake build
```

Comment:
add trace and verbosity to all stages - https://ruby.github.io/rake/doc/command_line_usage_rdoc.html